src/main/java/core/basesyntax/model/TransactionDto.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax.model;
+
+import java.util.Objects;
+
+public class TransactionDto {
+    private OperationType operationType;
+    private Fruit fruitName;
```

Comment:
Let's rename the field to **fruit**. We use **fruitName** like a String in Fruit